src/sdk/search/useSeachHistory.ts

```diff
@@ -0,0 +1,35 @@
+import { useStorage } from '@faststore/sdk'
+
+const storageKey = 'main::store::searchHistory'
+
+const MAX_HISTORY_SIZE = 5
```

Comment:
Should we move it to the `constants.ts`? 🤔

Reply:
Hmm nice question! Since it's something localised here, and not using somewhere else, maybe it's okay to leave it here 🤔

Reply:
That's true. Just thinking out loud :)